raw-spans-grouper/helm/templates/raw-spans-grouper-config.yaml

```diff
@@ -6,44 +6,50 @@ metadata:
     release: {{ .Release.Name }}
 data:
   application.conf: |-
-
     kafka.streams.config = {
+      # Core configs
       application.id = raw-spans-to-structured-traces-grouping-job
+      bootstrap.servers = "{{ .Values.rawSpansGrouperConfig.kafkaStreamsConfig.bootstrapServers }}"
+      schema.registry.url = "{{ .Values.rawSpansGrouperConfig.kafkaStreamsConfig.schemaRegistryUrl }}"
+      value.subject.name.strategy = "io.confluent.kafka.serializers.subject.TopicRecordNameStrategy"
+      # Core configs - For applications with state
+      num.stream.threads = "{{ int .Values.rawSpansGrouperConfig.kafkaStreamsConfig.numStreamThreads }}"
+      commit.interval.ms = "{{ int .Values.rawSpansGrouperConfig.kafkaStreamsConfig.commitIntervalMs }}"
       group.instance.id = ${?POD_NAME}
+      cache.max.bytes.buffering = "{{ int .Values.rawSpansGrouperConfig.kafkaStreamsConfig.cacheMaxBytesBuffering }}"
+      # Common client (prodcuer, consumer, admin) configs
+      receive.buffer.bytes = "{{ int .Values.rawSpansGrouperConfig.kafkaStreamsConfig.receiveBufferBytes }}"
+      send.buffer.bytes = "{{ int .Values.rawSpansGrouperConfig.kafkaStreamsConfig.sendBufferBytes }}"
+      # Producer configs
+      producer.acks = "{{ .Values.rawSpansGrouperConfig.kafkaStreamsConfig.producerAcks }}"
+      producer.batch.size = "{{ int .Values.rawSpansGrouperConfig.kafkaStreamsConfig.producerBatchSize }}"
+      producer.linger.ms = "{{ int .Values.rawSpansGrouperConfig.kafkaStreamsConfig.producerLingerMs }}"
+      producer.compression.type = "{{ .Values.rawSpansGrouperConfig.kafkaStreamsConfig.producerCompressionType }}"
+      producer.max.request.size = "{{ int .Values.rawSpansGrouperConfig.kafkaStreamsConfig.producerMaxRequestSize }}"
+      producer.buffer.memory = "{{ int .Values.rawSpansGrouperConfig.kafkaStreamsConfig.producerBufferMemory }}"
+      # Consumer configs
+      consumer.max.partition.fetch.bytes = "{{ int .Values.rawSpansGrouperConfig.kafkaStreamsConfig.consumerMaxPartitionFetchBytes }}"
+      consumer.max.poll.records = "{{ int .Values.rawSpansGrouperConfig.kafkaStreamsConfig.consumerMaxPollRecords }}"
+      consumer.session.timeout.ms = "{{ int .Values.rawSpansGrouperConfig.kafkaStreamsConfig.consumerSessionTimeoutMs }}"
+      # Changelog topic configs
+      replication.factor = "{{ int .Values.rawSpansGrouperConfig.kafkaStreamsConfig.replicationFactor }}"
+      topic.cleanup.policy = "delete,compact"
```

Comment:
I always wonder shouldn't be `compact,delete`? what does `delete,compact` mean?

Reply:
`compact,delete` `delete,compact` both are same.
It's basically a list of values. Order doesn't matter.